.gitignore

```diff
@@ -43,4 +43,7 @@ package-lock.json
 .env
 
 # vscode config file
-.vscode
\ No newline at end of file
+.vscode
+
+# Eslint cache 
+.eslintcache
```

Comment:
@nkpremices just a nitpick: This red symbol means you should add a new line to the end of the file. It's showing in several places, so just take a look at the PR and add a newline wherever this symbol is showing.

Reply:
I'm on it. Thank you